src/Ingredients/DisplayRecipes.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+
+namespace BreadAndIfit\Ingredients;
+
+
+class DisplayRecipes
+{
+    /**
+     *takes the JSON data returned from the API request and returns a string with HTML markup
+     *
+     * @param $json API response in JSON format
+     * @return string relevant recipes based on user selected ingredients or a error message string
+     */
+    public static function outputRecipes($json)
+    {
+        $recipes = json_decode($json)->results;
+
+        $validate = self::validateRecipe($recipes);
+
+        if ($validate) {
```

Comment:
put the method call in the if

Reply:
Done, resolved this by separating out the methods...